src/foraging_gui/Foraging.py

```diff
@@ -208,6 +210,35 @@ def _show_disk_space(self):
         else:
             self.DiskSpaceProgreeBar.setStyleSheet("QProgressBar::chunk {background-color: green;}")
 
+    def _load_stage(self) -> None:
+        """
+        Check whether newscale stage is defined in the config. If not, initialize and inject stage widget.
+        """
+        if self.Settings['newscale_serial_num_box{}'.format(self.box_number)] == '':
+            widget_to_replace = "motor_stage_widget" if self.default_ui == "ForagingGUI_Ephys.ui" else "widget_2"
+            self._insert_stage_widget(widget_to_replace)
+        else:
+            self._GetPositions()
+
+    def _insert_stage_widget(self, widget_to_replace: str) -> None:
+        """
+        Given a widget name, replace all contents of that widget with the stage widget
+        Note: The UI file must be loaded or else it can't find the widget to replace
+              Also the widget to replace must contain a layout so it can hide all child widgets properly.
+        """
+        logging.info("Inserting Stage Widget")
+
+        # Get QWidget object
+        widget = getattr(self, widget_to_replace, None)
+
+        if widget is not None:
+            layout = widget.layout()
+            # Hide all current items within widget being replaced
+            for i in reversed(range(layout.count())):
+                layout.itemAt(i).widget().setVisible(False)
+            # Insert new stage_widget
+            layout.addWidget(get_stage_widget())
```

Comment:
Could we make the new stage widget an attribute so it's easy to access if we need it? I'll probably need to use it to move the stage for the autolick port feature. Looking through you widget, I think going through the widget would be the only way to move the stage since the StageModel and device is embedded within. Is that right and would there be any issues with doing that?